### ISSUE TITLE
Attribute error if file does not exist

### ISSUE THREAD

**Issue by rendaw** [mosquito/aiofile#9]
Source (opening a nonexistent file):
```python
import aiofile
import asyncio

loop = asyncio.get_event_loop()


async def main():
    try:
        async with aiofile.AIOFile('x.txt', 'r', loop):
            pass
    except:
        pass


loop.run_until_complete(main())
```

I'd expect this to close normally with no error, but I get this:

```
Exception ignored in: <bound method AIOFile.__del__ of <AIOFile: 'x.txt'>>
Traceback (most recent call last):
  File ".../lib/python3.6/site-packages/aiofile/aio.py", line 74, in __del__
    self.close()
  File ".../lib/python3.6/site-packages/aiofile/aio.py", line 58, in close
    if self.__fileno == -2:
AttributeError: _AIOFile__fileno
```

**Comment by rendaw** [comment]
Sorry, this fails with the same error without the context manager actually:
```
import aiofile
import asyncio

loop = asyncio.get_event_loop()


async def main():
    try:
        aiofile.AIOFile('x.txt', 'r', loop)
    except:
        pass


loop.run_until_complete(main())
```

**Comment by mosquito** [comment]
Fixed on `0.5.3`
